scripts/dudle.coffee

```diff
@@ -192,16 +181,13 @@ parse_totals = (body) ->
     summary_rows = $('#summary').children()
     summary_items = elements_of_column summary_rows
 
-    totals = {}
+    totals = new Map()
     for i in [0..header_items.length - 1]
-        totals[header_items[i]] = summary_items[i]
+        totals.set(header_items[i], summary_items[i])
     totals
 
 elements_of_column = (row) ->
-    elements = []
-    for column in row.slice 1, row.length - 1
-        elements.push column.children[0].data
-    elements
+    row.slice(1, row.length - 1).map((column) -> column.children[0].data)
```

Comment:
I'm getting the following here `ERROR TypeError: Cannot read property '0' of undefined`